roles/ruby_s/tasks/install_from_source.yml

```diff
@@ -60,17 +60,6 @@
   ansible.builtin.shell: cd {{ install_path }}/{{ ruby_version }} && make install
   become: true
 
-- name: ruby_s | uninstall wrong version of bundler
```

Comment:
why are we removing this?

Reply:
Because it does not work anywhere near consistently, and does not report that it does not work, and in the rare cases it does work it makes the role fail idempotence